src/property-filter/__integ__/async-loading.test.ts

```diff
@@ -58,6 +58,9 @@ class AsyncPropertyFilterPage extends BasePageObject {
     await this.openValueEdit();
     await this.keys(value);
   };
+  clearFilteringInput = async () => {
```

Comment:
Would it be possible to use a unit test instead of the integ test?

Reply:
events after the interactive part didn't work well with the unit test for some reason

Reply:
I added some tests, all seems to be working: https://github.com/cloudscape-design/components/pull/2917